index.html

```diff
@@ -0,0 +1,189 @@
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script defer async type="module" src="index.js"></script>
+  </head>
+
+  <body class="flex wrapper">    
+  	<main>		
+  			<header>
+          
+          <!-- Desktop Pre Navigation bar-->
+
+  				<div class="pre-nav sub-section-grey">
+            <ul><a href="#"><i class="fa fa-facebook"></i></a></ul>
+            <ul><a href="#"><i class="fa fa-twitter"></i></a></ul>
+            <ul><a href="#" class="sub-section-text-white">English</a></ul>
+            <ul><a href="#" class="sub-section-text-white">My Page</a></ul>
+            <ul><a href="#" class="sub-section-text-white">Logout</a></ul>
+          </div>
+
+          <!-- Mobile Navigation Bar-->
+
+          <nav class="mobile-nav flex">
+            <div id="menu-bar" data-menu>
+              <div class="menu-bar"></div>
+              <div class="menu-bar"></div>
+              <div class="menu-bar"></div>
+            </div>
+            <a href="index.html"><img class="logo-mobile" src="images/logo_001.png" alt=""></a>
+          </nav>
+
+          <!-- Desktop Naviagtion Bar-->
+
+          <nav class="desktop-nav flex display-none">
+            <a href="#"><img class="logo" src="images/logo_001.png" alt="ISHWT 2023 Logo"></a>
+            <ul class="flex">
+              <li><a href="about.html">About</a></li>
+              <li><a href="#program-section">Program</a></li>
+              <li><a href="#speakers-section">Join</a></li>
+              <li><a href="#partners">Sponsors</a></li>
+              <li><a href="about.html/#see-past-header">News</a></li>
+              <li><a href="#">
+                <div class="campaign">CHG campaign</div>
+              </a></li>
+            </ul>
+          </nav>
+
+          <!-- Main Header-->
+
+          <section class="headline-section">
+            <p class="pre-header">"Hello! Health & Wellness World"</p>
+            <h1 class="main-title">International Summit on<br>Health & Wellness Tourism<br>2023</h1>
+            <p class="main-paragraph grey-text mobile-font">A joyful celebration believing in the value of Health and Wellness, creating a positive change with people from over 80 countries is taking place in October, in India.<br><br>A joyful celebration believing in the value of Health and Wellness, creating a positive change with people from over 80 countries is taking place in October, in India.</p>
+            <div class="program-date">
+              <p id="bold-text" class="mobile-font">2023.10.12(THU) ~ 18(WED)</p>
+              <p class="venue grey-text mobile-font">@ India Habitat Center, Jor Bagh, New Delhi, India</p>

```

Comment:
- [ ] Kindly fix this little mistake. 👇 
<img width="516" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235199486-e2dc538e-ab69-400f-8708-1266441495a4.png">

## Behance design
![Uploading Untitled.png…]()

Reply:
Done